app/javascript/pages/failed-import/component.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { PureComponent } from 'react';
+
+import Icon from 'components/ui/icon';
+import treeImage from 'assets/icons/error.svg';
+
+import './styles.scss';
+
+class ErrorPage extends PureComponent {
```

Comment:
Is this the old error page?

Reply:
yes. I am going to refactor to a reusable component shortly. For now its good enough I think